### PR TITLE
Fix Jaeger console example dotnet run command

### LINKED ISSUE #3953: Test Jaeger exporter example dotnet run command malformed
# Bug Report

Runtime version:

net6.0

## Symptom

The current comments in the `TestJaegerExporter` Run() method, state:

`
dotnet run jaeger -h localhost -p 6831
`

**What is the expected behavior?**

I expect to start the test jaeger exporter. This can be done with `dotnet run --project Examples.Console.csproj -- jaeger -h localhost -p 6831`. When running that command the command line output shows: "_Traces are being created and exported to Jaeger in the background. Use Jaeger to view them. Press ENTER to stop_."

**What is the actual behavior?**

If I run `dotnet run jaeger -h localhost -p 6831` in the command line, I see command line help.

## Reproduce

Change directories to the examples\console project and run `dotnet run jaeger -h localhost -p 6831` in the command line. You will see command line help for dotnet run instead of starting the test jaeger exporter.

## Additional Context

I am creating this issue so that I can reference it in the PR that will fix it. I am planning to make the fix shortly.

### PR DESCRIPTION
**Pull request by moonbox3**
Fixes #3953 

## Changes

Fix the Jaeger console example dotnet run command. Add comment on how to access localhost Jaeger UI.

### CODE REVIEW

File: examples/Console/TestJaegerExporter.cs [hunk 1]
```diff
@@ -45,9 +45,10 @@ internal static object Run(string host, int port)
             // the reporoot\examples\Console\.
             // (eg: C:\repos\opentelemetry-dotnet\examples\Console\)
             //
-            // dotnet run jaeger -h localhost -p 6831
+            // dotnet run --project Examples.Console.csproj -- jaeger -h localhost -p 6831
```
Comment (reyang) [inline examples/Console/TestJaegerExporter.cs:48]: Would this work?

```suggestion
            // dotnet run -- jaeger -h localhost -p 6831
```

Comment (moonbox3) [reply examples/Console/TestJaegerExporter.cs:48]: @reyang it does work as well, yes. It seems appropriate to specify the project. Is there any issue with specifying the project as is done [here](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/examples/Console/Program.cs#:~:text=///%20dotnet%20run%20%2D%2Dproject%20Examples.Console.csproj%20%2D%2D%20jaeger%20%2Dh%20localhost%20%2Dp%206831)?

Comment (reyang) [reply examples/Console/TestJaegerExporter.cs:48]: I don't have strong opinion here, personally I prefer shorter/simpler version if the intention is clear (e.g. I feel repeating the project name here is a duplicated information given the project folder already provided the context).

Not a blocker though.

**Review by reyang — APPROVED** [review]
LGTM.

**Pull request merged**